asym_crypto_yaml/crypto.py

```diff
@@ -0,0 +1,148 @@
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric import padding, rsa
+from cryptography.hazmat.primitives import hashes
+import yaml
+import base64
+
+# See https://cryptography.io/en/latest/
+
+def encode(s):
+    return base64.encodebytes(s).decode('ascii')
+
+def decode(b):
+    return base64.decodebytes(b.encode('ascii'))
+
+
+class Encrypted(str):
+     def __new__(self, value):
+        return str.__new__(self, value)
+     def __repr__(self):
+         return "!Encrypted %s " % self
+
+def encrypted_constructor(loader, node):
+     value = loader.construct_scalar(node)
+     return Encrypted(value)
+
+def encrypted_representer(self, data):
+    return self.represent_scalar('!Encrypted', data, style='|')
+
+def encrypt_value(str_input, public_key):
+    return Encrypted(encode(public_key.encrypt(
+        str_input.encode('utf-8'),
+        padding.OAEP(
+            mgf=padding.MGF1(algorithm=hashes.SHA256()),
+            algorithm=hashes.SHA256(),
+            label=None
+        )
+    )))
+
+def decrypt_value(input, private_key):
+    if(type(input) == Encrypted):
```

Comment:
No need for parens on an if-statement, and `isinstance(input, Encrypted)` is a better way to check types.

Reply:
I do not mind a drive by review at all, thank you!